core/src/test/java/apoc/convert/ConvertJsonTest.java

```diff
@@ -42,6 +42,79 @@
 	    testCall(db, "RETURN apoc.convert.toJson({a:42,b:\"foo\",c:[1,2,3]}) as value",
 	             (row) -> assertEquals("{\"a\":42,\"b\":\"foo\",\"c\":[1,2,3]}", row.get("value")) );
     }
+
+    @Test
+    public void testToJsonNode() throws Exception {
+	    testCall(db, "CREATE (a:Test {foo: 7}) RETURN apoc.convert.toJson(a) AS value",
```

Comment:
non compariamo le stringe perché le chiavi potrebbero cambiare ordine casulmente.
Definisci delle mappe per comparazione e trasforma le string ritornate in mappa

stessa cosa per gli esempi sotto